### PR TITLE
add Quick Brick protocol

### PR DESCRIPTION
**Pull request by f-roland**
This PR adds a protocol for the QBAppMananger app delegate class to provide pods access to quick brick zapp data

I chosed to create a new protocol instead of extending an existing one for better separation of concerns, and to avoid implementing unnecessary methods in the QBAppManager

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Gemfile
+source 'https://rubygems.org'
+
+gem 'fastlane'
+gem 'xcpretty'
+gem 'cocoapods', '= 1.5.2'
```
Comment (f-roland) [inline Gemfile]: I'd use `~> 1.5.2` to avoid a crash I just had on CI on another repo

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      - DEPLOY_VERSION: "0.0.24"
+      - DEPLOY_COMMAND: "~/.scripts-iOS/circleCi/deploySingleFramework.sh"
+      #fastlane variables
+      - LC_ALL: "en_US.UTF-8"
+      - LANG: "en_US.UTF-8"
+    working_directory: ~/ZappAppConnector-iOS
+    steps:
+      - run:
+          name: Ruby - set version
+          command:  echo "ruby-2.4" > ~/.ruby-version
```
Comment (simonvaucher) [inline .circleci/config.yml]: 100% sure about that? 2.4 is not too cutting edge?

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+      - run:
+          name: Ruby - version
+          command:  ruby -v
+
+      # Prepere Git and Submodules
```
Comment (simonvaucher) [inline .circleci/config.yml]: Prepare

File: .circleci/config.yml (hunk 2 above)
Comment (f-roland) [reply .circleci/config.yml]: this is the default version installed on ci macos machines - using anything else would seriously increase build time

**Pull request merged**